sip-quickstart/README.md

```diff
@@ -0,0 +1,69 @@
+# sip-quickstart
+
+Creates and wires up a new SIP domain for incoming and outgoing SIP calls.
+
+## Pre-requisites
+
+### Environment variables
+
+This project requires some environment variables to be set. To keep your tokens and secrets secure, make sure to not commit the `.env` file in git. When setting up the project with `twilio serverless:init ...` the Twilio CLI will create a `.gitignore` file that excludes `.env` from the version history.
+
+In your `.env` file, set the following values:
+
+| Variable | Description | Required |
+| :------- | :---------- | :------- |
+
+
+### Function Parameters
+
+`/blank` expects the following parameters:
```

Comment:
These functions no longer exist.

It would be useful to have some documentation of the functions this provides. If you could describe which functions receive webhooks from Twilio, that would probably be better than just describing the parameters that are required.

Reply:
Doh! Forgot to change the default README... will get this in there!